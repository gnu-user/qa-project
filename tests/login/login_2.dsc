TITLE
    Login Account File
DESCRIPTION
    Attempt to initiate the login transaction by entering “login” into the 
    input, and then read in the current user accounts file. Verify that the 
    login account file has been read by logging in using an account read from 
    the file.
DEPENDENCIES
