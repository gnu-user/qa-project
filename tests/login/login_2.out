Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
> 